dbt_project.yml

```diff
@@ -25,6 +25,7 @@ clean-targets: # directories to be removed by `dbt clean`
 
 vars:
   truncate_timespan_to: "{{ current_timestamp() }}"
+  jaffle_raw_schema: "{{ env_var('JAFFLE_RAW_SCHEMA', 'jaffle_raw'}}) }}"
```

Comment:
Previously this would have been 'analytics_raw'.

Since the files are source with the meta->external_location instruction, this shouldn't cause any breakage.